webpack/components/PreupgradeReportsList/components/PreupgradeReportEntry.js

```diff
@@ -1,16 +1,64 @@
 import React from 'react';
-import { ListView } from 'patternfly-react';
+import { ListView, Grid, Icon } from 'patternfly-react';
 import PropTypes from 'prop-types';
 
-import { additionalInfo } from './helpers';
+import { presentSeverity, tagInfo, hasRemediations } from './helpers';
 
 const PreupgradeReportEntry = ({ entry }) => (
   <ListView.Item
     key={entry.id}
     className="listViewItem--listItemVariants"
     stacked
-    additionalInfo={additionalInfo(entry)}
-  />
+    description={entry.title}
+    additionalInfo={[
+      <ListView.InfoItem key="k-host">
```

Comment:
`key` prop can be removed, I think it is needed only when using component inside iterator.

Reply:
I'm still getting these error, and can't find where is it.. any clue?

```Warning: Each child in a list should have a unique "key" prop.

Check the render method of `PreupgradeReportEntry`. See https://fb.me/react-warning-keys for more information.
    in div (created by PreupgradeReportEntry)
    in PreupgradeReportEntry (created by PreupgradeReportsList)
    in div (created by ListView)
    in ListView (created by PreupgradeReportsList)
    in PreupgradeReportsList (created by PreupgradeReports)
    in LoadingState (created by PreupgradeReports)
    in PreupgradeReports (created by WrappedPreupgradeReports)
    in WrappedPreupgradeReports (created by I18nProviderWrapper(WrappedPreupgradeReports))
    in IntlProvider (created by I18nProviderWrapper(WrappedPreupgradeReports))
    in I18nProviderWrapper(WrappedPreupgradeReports) (created by StoreProvider(I18nProviderWrapper(WrappedPreupgradeReports)))
    in Provider (created by StoreProvider(I18nProviderWrapper(WrappedPreupgradeReports)))
    in StoreProvider(I18nProviderWrapper(WrappedPreupgradeReports)) (created by DataProvider(StoreProvider(I18nProviderWrapper(WrappedPreupgradeReports))))
    in DataProvider(StoreProvider(I18nProviderWrapper(WrappedPreupgradeReports)))```

Reply:
My guess would be `getExternals` method as the returned markup items created in iterator do not have `key` property.